src/main/java/org/alfresco/repo/node/AbstractNodeServiceImpl.java

```diff
@@ -1,28 +1,28 @@
-/*
```

Comment:
There are a lot of whitespace changes in this file, can't we just revert it back to its original format?

Reply:
I have reverted the whitespace changes in this file.